Jenkinsfile

```diff
@@ -261,9 +261,11 @@ def doBuildWindows(String version, boolean isPublishingRun) {
         node('windows') {
             getArchive()
             try {
-              bat "\"${tool 'msbuild'}\" \"Visual Studio\\Realm.sln\" /p:Configuration=Debug /p:Platform=\"Win32\""
-              bat "\"${tool 'msbuild'}\" \"Visual Studio\\Realm.sln\" /p:Configuration=\"Static lib, release\" /p:Platform=\"Win32\""
-              bat "\"${tool 'msbuild'}\" \"Visual Studio\\Realm.sln\" /p:Configuration=\"Static lib, debug\" /p:Platform=\"Win32\""
+              ['Win32', 'x64'].each { platform ->
```

Comment:
these functional Groovy construct should be avoided in Jenkinsfiles since they require an iterator, which is not serializable, which does not play well with the way Jenkins operates remoting. In this case unrolling seems to be the easiest option, otherwise using for loops with indices is the way to go.